css/style.css

```diff
@@ -71,6 +71,7 @@ code {
 	border-radius: 3px;
 	font-size: 0.9em;
 	padding: 2px 4px;
+	word-break: break-word;
```

Comment:
This doesn't work on Firefox (and others) because `break-word` is non-standard. Is there anything else we could use?

Reply:
TIL.
```suggestion
	word-break: break-all;
```